Pod/Classes/Exp.swift

```diff
@@ -363,10 +363,10 @@ public func getContentNode(uuid:String) -> Promise<ContentNode>{
  @param dictionary of search params
  @return Promise<SearchResults<ContentNode>>.
  */
-public func findContentNodes(params:[String:AnyObject]) -> Promise<SearchResults<ContentNode>>{
+public func findContent(params:[String:AnyObject]) -> Promise<SearchResults<Content>>{
```

Comment:
we should make this backwards compatible, it should support: getContent(), getContentNode(), findContent(), findContentNodes()